src/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,87 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    @include interText(700, 16px, 19px, $black);
+}
+
+.dropdown {
+    all: unset;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 56px;
+    padding: 16px;
+    margin-bottom: 8px;
+    background: $gray;
+    border-radius: 8px;
+    box-sizing: border-box;
+
+    &:active {
+        background: $black ;
+    }
+
+    &:hover {
+        background: #E6E6E6;
```

Comment:
Why this is not a variable?

Reply:
this color was used in one place only. So there was no reason for me to make this a variable. Should I make this a variable anyway?

Reply:
For the code integrity, I would recommend to do so.